docs/getting-started-logging.md

```diff
@@ -56,19 +56,34 @@ We will deploy the following components on a Kubernetes cluster as follows.
    helm install scalar-logging-loki grafana/loki-stack -n monitoring -f scalar-loki-stack-custom-values.yaml
    ```
 
-## Step 3. Access the Grafana dashboard
+## Step 3. Add a Loki data source in the Grafana configuration
+
+1. Add a configuration of the Loki data source in the `scalar-prometheus-custom-values.yaml` file.
+   ```yaml
+   grafana:
+     additionalDataSources:
+     - name: Loki
+       type: loki
+       uid: loki
+       url: http://scalar-logging-loki.monitoring.svc.cluster.local:3100/
```

Comment:
```suggestion
       url: http://scalar-logging-loki:3100/
```

Reply:
Ah, sorry. I overlooked that part...
Thank you for pointing it out!